pkgs/development/python-modules/palace/alure2-config.diff

```diff
@@ -0,0 +1,11 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 4c9aedb..f3ed424 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -6,5 +6,5 @@ get_target_property(include_dirs Alure2::alure2 INTERFACE_INCLUDE_DIRECTORIES)
+ message("alure2_include_dirs=${include_dirs}")
+ 
+ get_target_property(link_libraries Alure2::alure2 INTERFACE_LINK_LIBRARIES)
+-get_target_property(libalure2 Alure2::alure2 IMPORTED_LOCATION_NOCONFIG)
++get_target_property(libalure2 Alure2::alure2 IMPORTED_LOCATION_RELEASE)
+ message("alure2_extra_objects=${link_libraries};${libalure2}")
```

Comment:
Could this not have been done with `substituteInPlace`?

Reply:
Thanks for the suggestion.